experimental/torch_xla2/torch_xla2/ops/jaten.py

```diff
@@ -2043,6 +2042,10 @@ def _aten_frexp(input):
 # aten.gather
 @op(torch.ops.aten.gather)
 def _aten_gather(input, dim, index):
+  if input.ndim == 0:
```

Comment:
Just curious, how did you debug the gather implementation? I also had traced `quantile` problems to the `gather` implementation but had not yet started investigating further. I was going to compare it to other gather implementations in jax/pytorch as a starting point, what did you end up doing?